msteams-platform/m365-apps/publish.md

```diff
@@ -36,7 +36,7 @@ As a global admin, you can upload and pre-install the app package from [Microsof
 
 ## Multitenant distribution
 
-The [Microsoft AppSource](https://appsource.microsoft.com/) (Microsoft commercial marketplace) submission process for Teams apps enabled for Outlook and Office is same as for traditional Teams apps. The only difference is that you need to use Teams app manifest [version 1.13](../tabs/how-to/using-teams-client-sdk.md) in your app package, which introduces support for Teams apps that run across Microsoft 365.
+The [Microsoft commercial marketplace](https://appsource.microsoft.com/) (Microsoft AppSource) submission process for Teams apps enabled for Outlook and Office is same as traditional Teams apps. The only difference is you'll need to use Teams app manifest [version 1.13](../tabs/how-to/using-teams-client-library.md) in your app package, which introduces support for Teams apps that run across Microsoft 365.
```

Comment:
The website title is **Microsoft AppSource**, please check if the link text should be retained as earlier.

Reply:
Good callout--I double-checked the content using Acrolinx tool, and the branding guidance is indeed to use "the Microsoft commercial marketplace" wherever possible (and disambiguate using the website name, Microsoft AppSource).